samples-check/src/test/samples/gradle/composite-sample/basic/compositeBuildsBasicCli.sample.out

```diff
@@ -13,6 +13,5 @@
 > Task :run
 The answer is 42
 
-
 BUILD SUCCESSFUL in 0s
-2 actionable tasks: 2 executed
+6 actionable tasks: 6 executed
```

Comment:
Where included builds not counted previously?

Reply:
it looks like - shall I investigate it or we're good?